clang/lib/Frontend/CompilerInstance.cpp

```diff
@@ -822,6 +822,7 @@ CompilerInstance::createOutputFileImpl(StringRef OutputPath, bool Binary,
   Expected<std::unique_ptr<OutputFile>> O =
       getOrCreateOutputBackend().createFile(
           OutputPath, OutputConfig()
+                          .setText(!Binary)
```

Comment:
Can you add a FIXME? Seems like `setTextWithCRLF` should be sufficient.

Reply:
Or maybe we define output config to have separate `Text` and `CRLF` field. `.setTextWithCRLF` can be a convenience method in `OutputConfig` to set both.

Reply:
Yes, that's what I did over at https://github.com/dexonsmith/llvm-project/commit/21c24f6866048a2f580ffd5cc633c6eed8861862 — a FIXME for now would probably be enough though.

Reply:
Updated to also have `setBinary()` and `getBinary()` -- https://github.com/dexonsmith/llvm-project/commit/f037dc0544b6efaff70c206e922a1db4978742bd#diff-0598fa12397ba618b927970836f454d312e68b633ebb634484e7770f0ab9177bR45 -- will send for review soon